package.json

```diff
@@ -1,14 +1,15 @@
 {
   "scripts": {
-    "prestart": "source $HOME/.nvm/nvm.sh; nvm use 18.17.0",
-    "start": "npx expo start",
+    "prestart": "chmod +x prestart.sh && ./prestart.sh",
```

Comment:
on se pose la question avec Manon de garder la ligne prestart, même côté mobile. En effet au départ j'avais rajouté engines node 18.17.0 (voir plus plus bas) que j'ai dû changer en engines node ^18.17.0 car le build dev Expo échouait car je cite "The engine "node" is incompatible with this module. Expected version "18.17.0". Got "18.18.0"". Donc apparemment expo utilise node 18.18.0 et non node 18.17.0. Et ce depuis longtemps au regard des anciens builds. Ca me fait me demander donc si cette ligne est vraiment prise en compte puisque du coup node 18.17.0 n'est pas utilisé. (pour info en la supprimant, cela marche toujours en local et sur un build dev Expo)

Reply:
as tu creuser le fonctionnement de la commande `nvm use` : est ce qu'elle prend la version brute ou une equivalence ?

Reply:
nvm use prend la version brute. Mais j'ai l'impression que start et prestart ne sont pas utilisée pour lancer un build, que ces commandes ne sont utilisées qu'en local (https://docs.expo.dev/build-reference/npm-hooks/)